auth/import_users.go

```diff
@@ -143,8 +143,19 @@ func (u *UserToImport) PhoneNumber(phoneNumber string) *UserToImport {
 
 // Metadata setter.
 func (u *UserToImport) Metadata(metadata *UserMetadata) *UserToImport {
-	u.set("createdAt", metadata.CreationTimestamp)
-	return u.set("lastLoginAt", metadata.LastLogInTimestamp)
+	if metadata == nil {
```

Comment:
I wonder if we should just let the API panic in case somebody passes `nil`. Silently ignoring `nil` argument somehow feels wrong. WDYT?